### PR TITLE
Document vulkan implementation details

### PR DESCRIPTION
**Pull request by aclysma**

### CODE REVIEW

File: docs/api/backends/vulkan.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+features that are not yet implemented, see Future Work.
+
+## Platform Support
+
+Detailed info here: https://vulkan.gpuinfo.org
+
+Vulkan is well supported on a wide variety of desktop hardware and operating systems.
+* [AMD](https://www.amd.com/en/technologies/vulkan) - R5 240, 500 series, etc. and up (~circa 2013)
+* [Intel](https://www.intel.com/content/www/us/en/support/articles/000005524/graphics.html) - Skylake/HD 500 or later (~circa 2015)
+* [NVIDEA](https://developer.nvidia.com/vulkan-driver) - Kepler/600 series and up (~circa 2012)
```
Comment (DavidVonDerau) [inline docs/api/backends/vulkan.md:24]: Typo ` Nvidia`

File: docs/index.md [hunk 2]
```diff
@@ -19,6 +19,11 @@
     * [Resource Binding Model](api/resource_binding_model.md)
     * [Validation and Debugging](api/validation_and_debugging.md)
     * [API Triangle Example](../rafx/examples/api_triangle/api_triangle.rs)
+    * Backend Implementation Details:
+        * [Vulkan](api/backends/vulkan.md)
+        * [Metal](api/backends/metal.md)
+        * [GL ES 2.0](api/backends/gles2.md)
+        * GL ES 3.0 (TODO)
```
Comment (DavidVonDerau) [inline docs/index.md:26]: Should I hold off on approving until this last doc is written?

File: docs/api/backends/gles2.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+OpenGL has logging that can be enabled on desktop OpenGL systems (but is not technically part of the base standard.)
+This was not added to OpenGL ES until version 3.2. Some feedback is also available when running in a web browser.
+
+This functionality is not properly exposed in the public API yet.
+
+### Debugging
+
+These tools can be used to debug/trace a frame
+* Renderdoc
+* NVIDEA Nsight
```
Comment (DavidVonDerau) [inline docs/api/backends/gles2.md:54]: `Nvidia`

File: docs/api/backends/vulkan.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+### Validation
+
+By default, rafx-api will try to enable validation in debug mode. This requires installing the vulkan SDK. It is
+highly recommended that you install the SDK and run with validation on from time to time.
+
+### Debugging
+
+These tools can be used to debug/trace a frame
+ * Renderdoc
+ * NVIDEA Nsight
```
Comment (DavidVonDerau) [inline docs/api/backends/vulkan.md:50]: Find / replace `Nvidea` with `Nvidia`

File: docs/index.md (hunk 2 above)
Comment (aclysma) [reply docs/index.md:26]: If you approve it before I add it, I can do it as a separate PR.

**Review by DavidVonDerau — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -23,18 +23,16 @@ Rafx supports most mainstream platforms. Proprietary platforms can be supported
 | -------- | -------- | ----- | ------------- |
```
[the remark this answers is not in the document]
Comment (aclysma) [reply README.md:23]: Yeah, holding off on making it more visible until it's more polished :)

**Pull request merged**